src/charm.py

```diff
@@ -9,18 +9,19 @@
 from typing import Dict
 
 from charms.prometheus_k8s.v0.prometheus_scrape import MetricsEndpointProvider
+from charms.traefik_k8s.v1.ingress import IngressPerAppRequirer
 from ops.charm import CharmBase, HookEvent, WorkloadEvent
 from ops.main import main
 from ops.model import ActiveStatus, BlockedStatus, MaintenanceStatus, WaitingStatus
 
-from github_actions_exporter import GitHubActionsExporter
-from ingress import set_nginx_route
-from state import State
-from types_ import CharmState
+from charm_state import CharmState
+from github_actions_exporter import exporter_environment, exporter_version, is_configuration_valid
```

Comment:
Should the function names in the module not include things like `exporter` since that is already implied by the function being in the exporter module?

Reply:
I liked the "gh_exporter" name, makes sense. I moved the constants to github_actions_exporter file now.